src/Controls.js

```diff
@@ -74,10 +74,11 @@ var Controls = React.createClass({
         </select>{' '}
         <input ref='start' type='text' />–
         <input ref='stop' type='text' />{' '}
-        <button>Go</button>{' '}
-
-        <button onClick={this.zoomOut}>-</button>{' '}
-        <button onClick={this.zoomIn}>+</button>
+        <button className='btn-submit'>Go</button>{' '}
```

Comment:
Is the `btn` prefix a bootstrap thing? You can also style it using `button.submit` if it's important to make it clear that this is a button.